bedrock/firefox/templates/firefox/android/index.html

```diff
@@ -22,9 +23,13 @@
 {% block site_header %}{% endblock %}
 
 {% block site_header_unwrapped %}
-  {% call fxfamilynav('dark') %}
+  {% call fxfamilynav() %}
     <a class="dl-button" id="nav-dl-button" rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}">
-      {{ _('Download on Google Play') }}
+      {% if l10n_has_tag('android_spring_updates') %}
+        {{ _('Get Firefox for android') }}
```

Comment:
Android (uppercase). We already have the string in [main.lang](http://transvision.mozfr.org/string/?entity=mozilla_org/main.lang:8156a2ee&repo=mozilla_org), so no need for tag.